app/main.py

```diff
@@ -8,16 +8,19 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return f"Distance(km={self.km})"
 
+    # якщо вказувати Distance в анатації other
+    # зявляється помилка
+    # TypeError: unsupported
+    # operand type(s)
+    # for |: 'types.UnionType' and 'str'
+
     def __add__(self, other: int | float) -> "Distance":
         if isinstance(other, Distance):
             return Distance(self.km + other.km)
```

Comment:
but `other` can be a `Distance` class. So I believe that the error is caused because of wrong type annotation syntax